collect_app/src/main/java/org/odk/collect/android/spatial/MapHelper.java

```diff
@@ -256,5 +262,19 @@ public boolean accept(File dir, String filename) {
         });
     }
 
+    private boolean isFileFormatSupported(File file) {
```

Comment:
Can you please add a little comment here so we remember why this method is here more easily? Might just be "osmdroid doesn't currently support pbf tiles: https://github.com/osmdroid/osmdroid/issues/101"